### PR TITLE
🍆 Add Sik | crifferent.de

### PR DESCRIPTION
**Pull request by SimonKraus**

### CODE REVIEW

**Comment by SimonKraus** [comment]
https://kusama.subscan.io/extrinsic/16789369-2

**Comment by stepanLav** [comment]
Thank you for your request!
@SimonKraus 

You can check how it will look like:

<details>
  <summary>App screens</summary>

Screenshot 2023-03-01 at 13 06 49
Screenshot 2023-03-01 at 13 06 52
Screenshot 2023-03-01 at 13 06 55


</details>

File: registry/kusama.json [hunk 1]
```diff
@@ -30,5 +30,13 @@
...
+        "shortDescription": "DotSama native node operator and community supporter",
+        "longDescription": "## About Sik\nFounder of [Crifferent Staking Services](https://crifferent.de)<br/>Developer for [Staking Facilities](https://stakingfacilities.com)<br/>Member of [DOT Validator Alliance DAO](https://dotvalidators.org)<br/>Team member @ [HydraDX](https://hydradx.io)/[Basilisk](https://bsx.fi)<br/>RPC provider @ [UnitedBloc](https://unitedbloc.com/)<br/>Member of the [Infrastructure Builders Program](https://ibp.network/)\n\n---\n\n**From within the ecosystem - for the ecosystem!**<br/>_I'm evaluating proposals by taking different views into consideration. I'm not only looking at it from the perspective of a community member, but also as a developer/founder/infra-provider/human being._\n\n---\n## Contact me\n@dev0_sik @ [Twitter](https://twitter.com/dev0_sik)<br />@dev0_sik @[Telegram](https://t.me/dev0_sik)<br />sik#3706 @ [Discord](https://discord.com/)<br />@dev0_sik:matrix.org @ [Element](https://element.io/)<br />[simon.kraus@crifferent.de](mailto:simon.kraus@crifferent.de)<br />WhatsApp/Callboy +49 157 377 309 02",
```
Comment (stepanLav) [inline registry/kusama.json:39]:
```suggestion
        "longDescription": "## About Sik\nFounder of [Crifferent Staking Services](https://crifferent.de) \n\nDeveloper for [Staking Facilities](https://stakingfacilities.com) \n\nMember of [DOT Validator Alliance DAO](https://dotvalidators.org) \n\nTeam member @[HydraDX](https://hydradx.io)/[Basilisk](https://bsx.fi) \n\nRPC provider @[UnitedBloc](https://unitedbloc.com/) \n\nMember of the [Infrastructure Builders Program](https://ibp.network/)\n\n---\n\n**From within the ecosystem - for the ecosystem!** \n\nI'm evaluating proposals by taking different views into consideration. I'm not only looking at it from the perspective of a community member, but also as a developer/founder/infra-provider/human being._\n\n---\n## Contact me \n\n@dev0_sik @[Twitter](https://twitter.com/dev0_sik) \n\n@dev0_sik @[Telegram](https://t.me/dev0_sik) \n\nsik#3706 @[Discord](https://discord.com/) \n\n@dev0_sik:matrix.org @[Element](https://element.io/) \n\n[simon.kraus@crifferent.de](mailto:simon.kraus@crifferent.de) \n\nWhatsApp/Callboy +49 157 377 309 02",
```
I suggest to do longDescription by that syntax, in order to get that result:

<details>
  <summary>App screens</summary>

Screenshot 2023-03-01 at 14 51 00
Screenshot 2023-03-01 at 14 50 52

</details>

**Comment by SimonKraus** [comment]
Thanks @stepanLav 🤘

**Review by stepanLav — APPROVED** [review]

**Pull request merged**